upcase_booklist_app/templates/book_list.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Curious Programmer</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+    </head>
+    <body> 
+        <div class="page-header">
+            <p class="nav-item"><a href="{{ url_for('book_list') }}">Books</a></p>
+        </div>
+
+        <div class="page-title">         
+            <h1>Books</h1>
+        </div>
+
+        <div class="books-to-read">
+            {% for title, details in BOOKS.items() %} 
```

Comment:
Nice job with this `for` loop; looks solid.

Reply:
I'd recommend naming `BOOKS` as `books`, because it won't be a constant forever, as when you start adding, updating and deleting books